.github/workflows/tests.yml

```diff
@@ -17,6 +17,10 @@ jobs:
                     - php: '7.2'
                     - php: '7.3'
                     - php: '7.4'
+                    - php: '8.0'
+                    - php: '8.1'
+                    - php: '8.2'
+                    - php: '8.3'
```

Comment:
Try updating these:

`actions/checkout@v3` -> to v4

`actions/cache@v2` -> to v4

`php-actions/composer@v5` -> to v6

Reply:
Done